lib/runtime-core/image-loading-freebsd-x86-64.s

```diff
@@ -0,0 +1,127 @@
+# NOTE: Keep this consistent with `fault.rs`.
```

Comment:
I checked it visually, but it looks like it's a copy of the linux one. In your opinion should we just rename it to something like sysv-elf and use the same for both, or do you think there's something here that FreeBSD might change and need a different image loader for?

Reply:
Yes it's the same as the linux one. I think we can use the same for Linux/FreeBSD.